sys/vm/vm_glue.c

```diff
@@ -525,7 +525,8 @@ kstack_cache_init(void *null)
 	    atop(VM_MAX_KERNEL_ADDRESS - VM_MIN_KERNEL_ADDRESS));
 	vm_object_set_flag(kstack_object, OBJ_HASCAP);
 	kstack_cache = uma_zcache_create("kstack_cache",
-	    kstack_pages * PAGE_SIZE, NULL, NULL, NULL, NULL,
+	    (kstack_pages + KSTACK_GUARD_PAGES) * PAGE_SIZE,
```

Comment:
Upstreamable?

Reply:
Moved to the upstreamable PR  #1008 to keep track.